doc/api/figure_api.rst

```diff
@@ -5,5 +5,305 @@
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.savefig
+
+
+Annotating
+==========
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.colorbar
+   Figure.legend
+   Figure.text
+   Figure.suptitle
+   Figure.get_suptitle
+   Figure.supxlabel
+   Figure.get_supxlabel
+   Figure.supylabel
+   Figure.get_supylabel
+   Figure.align_labels
+   Figure.align_xlabels
+   Figure.align_ylabels
+   Figure.autofmt_xdate
+
+
+Figure geometry
+===============
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.set_size_inches
+   Figure.get_size_inches
+   Figure.set_figheight
+   Figure.get_figheight
+   Figure.set_figwidth
+   Figure.get_figwidth
+   Figure.dpi
+   Figure.set_dpi
+   Figure.set_dpi
+
+Subplot layout
+==============
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.subplots_adjust
+   Figure.set_layout_engine
+   Figure.get_layout_engine
+
+Discouraged or deprecated
+-------------------------
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.tight_layout
+   Figure.set_tight_layout
+   Figure.get_tight_layout
+   Figure.set_constrained_layout
+   Figure.get_constrained_layout
+   Figure.set_constrained_layout_pads
+   Figure.get_constrained_layout_pads
+
+
+Interacting
+===========
+
+.. seealso::
+
+   - :ref:`event-handling`
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.ginput
+   Figure.add_axobserver
+   Figure.waitforbuttonpress
+   Figure.pick
+
+Modifying appearance
+====================
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.set_frameon
+   Figure.get_frameon
+   Figure.set_linewidth
+   Figure.get_linewidth
+   Figure.set_facecolor
+   Figure.get_facecolor
+   Figure.set_edgecolor
+   Figure.get_edgecolor
+
+Adding and getting Artists
+==========================
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.add_artist
+   Figure.get_children
+   Figure.figimage
+
+Getting and modifying state
+===========================
+
+.. seealso::
+
+   - :ref:`interactive_figures`
+
+.. autosummary::
+   :toctree: _as_gen
+   :template: autosummary.rst
+   :nosignatures:
+
+   Figure.clear
+   Figure.gca
+   Figure.sca
+   Figure.get_tightbbox
+   Figure.get_window_extent
+   Figure.show
+   Figure.set_canvas
+   Figure.draw
+   Figure.draw_without_rendering
+   Figure.draw_artist
+
+.. _figure-api-subfigure:
+
+SubFigure
+=========
+
+Matplotlib has the concept of a `~.SubFigure`, which is a logical figure inside
+a parent `~.Figure`.  It has many of the same methods as the parent.  See
+:ref:`nested_axes_layouts`.
+
+.. plot::
+
+   fig = plt.figure(layout='constrained', figsize=(4, 2.5), facecolor='lightgoldenrodyellow')
```

Comment:
The width of the figure is fixed at approximately 1/2 the maximum width of the text, so it would have to be twice as wide to match the text width.  You must be looking at a thinner viewport. 
<img width="1529" alt="layout" src="https://github.com/matplotlib/matplotlib/assets/1562854/a2e66eea-0344-4834-801b-48110d63cb8f">